src/components/PopoverMenu/popoverMenuPropTypes.js

```diff
@@ -54,6 +57,7 @@ const propTypes = {
 const defaultProps = {
     animationIn: 'fadeIn',
     animationOut: 'fadeOut',
+    animationInTiming: 300,
```

Comment:
@Uros787 Just realised that this exist as a `CONST.ANIMATED_TRANSITION`. Can we please replace the hardcoded value this the `CONST` ?